backend/api/views/EventResultView.py

```diff
@@ -0,0 +1,31 @@
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status
+from api.models import MeetEvent, Heat
+from api.serializers.EventResultSerializer import EventResultSerializer
+from drf_spectacular.utils import extend_schema, OpenApiParameter
+from api.CustomFilter import filter_by_group
+
+
+
+@extend_schema(tags=['Results'])
+class EventResultView(APIView):
+    @extend_schema(parameters =[OpenApiParameter(name="group_id", type=int)],summary="Displays the results of an event")
+    def get(self, request, event_id):
+        #Get event instance
+        try:
+            event_instance = MeetEvent.objects.get(id=event_id)
+        except MeetEvent.DoesNotExist:
+            return Response({'error': 'Event not found'}, status=status.HTTP_404_NOT_FOUND)
+
+        #Return all heats record with event = even_instance order by heat_time
+        data = self.get_queryset(event_instance)
+        serializer = EventResultSerializer(data, many=True)
+        return Response(serializer.data)
```

Comment:
Do we want the results to be paginated?

Reply:
I was thinking on showing the whole list all the time (It should not be that long), what do you think?

Reply:
That's what I though. 
I was considering pagination to be consistent with the other endpoints and the pagination setting.
But I feel that the results could be displayed all at once regardless of the number of athletes.